hybrasyl/XML/Examples/items/Blouse.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<item xmlns="http://www.hybrasyl.com/XML/Items">
+  <name>Blouse</name>
+  <properties>
+    <appearance sprite="115" equip="-1" display="1" bodystyle="transparent"/>
+    <stackable max="1"/>
+    <physical value="0" weight="0" durability="0" perishable="false" vendorable="false" bound="false"/>
+    <equipment slot="armor" unique="false"/>
+    <stateffects>
+      <base str="0" con="0" int="0" wis="0" dex="0" hp="0" mp="0"/>
+      <combat hit="0" dmg="0" ac="0" regen="0" mr="0"/>
+      <elemental defense="0"/>
+    </stateffects>
+    <flags>depositable enchantable exchangeable</flags>
```

Comment:
Nit but do you think it's better to have flags as a set of strings vs
    <flags>
        <depositable />
        <enchantable />
        <exchangeable />
    </flags>

We could validate flags that way; don't think it matters too much though.